components/brave_search/browser/brave_search_host.cc

```diff
@@ -5,26 +5,90 @@
 
 #include "brave/components/brave_search/browser/brave_search_host.h"
 
-#include <utility>
```

Comment:
I think you still need that header for std::move

Reply:
Fixed via interactive rebase.